index.html

```diff
@@ -0,0 +1,1564 @@
+          <figure class="position-relative">
+            <img src="assets/expert1.png" alt="Image news" class="w-100" />
+          </figure>
+          <a class="h5 font-weight-bolder text-body" href="#">Newsweek Announces Launch of the Newsweek Expert Forum</a>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+
+        <div class="card border-0 mb-2 order-first">
+          <figure class="position-relative">
+            <img src="assets/expert2.jpg" alt="Image news" class="w-100" />
+          </figure>
+          <a class="h5 font-weight-bolder text-body" href="#">Top Workplace Trends to Anticipate in 2021</a>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+
+        <article class="border-bottom d-flex">
+          <div class="col-md-5 col-lg-3 col-3 p-0 pb-md-1 pb-0 h-md-50">
+            <img src="assets/expert3.jpg" alt="avatar" />
+          </div>
+          <div class="col-md-7 col-lg-9">
+            <a href="#" class="font-weight-bold text-body">Meet Six Founding Members of the Newsweek Expert Forum</a>
+          </div>
+        </article>
+
+        <article class="py-1 border-bottom d-flex">
+          <div class="col-md-5 col-lg-3 col-3 p-0 pb-md-1 h-md-50">
+            <img src="assets/expert4.jpg" alt="avatar" />
+          </div>
+          <div class='col-md-7 col-lg-9'>
+            <a href="#" class="font-weight-bold text-body">Get to Know Six Members of Newsweek Expert Forum
+            </a>
+          </div>
+        </article>
+
+        <article class="pt-1 d-flex">
+          <div class="col-md-5 col-lg-3 col-3 p-0 pb-md-1 h-md-50">
+            <img src="assets/expert5.jpg" alt="avatar" />
+          </div>
+          <div class="col-md-7 col-lg-9">
+            <a href="#" class="font-weight-bold text-body"> Leaders from Newsweek Expert Forum
+              Meet Six Leaders from Newsweek Expert Forum
+            </a>
+          </div>
+        </article>
+      </div>
+    </div>
+  </section>
+
+
+  <section class="row border-bottom bg-white py-5">
+
+    <div class="col-lg-8 mb-md-3 my-5 m-lg-0">
+      <h6 class="text-danger">SUBSCRIBE</h6>
+      <div class="d-sm-flex bg-lightgray h-100 bg-light">
+        <div class="col-lg-5 col-sm-4 col-12">
+          <img class="mt-3 ml-3" src="assets/cover.png" alt="Subscription image">
+        </div>
+        <div class="ml-sm-3 col-sm-6 my-auto text-center text-sm-left">
+          <h4 class="font-weight-bold">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h4>
+          <p class="font-weight-lighter">All Access + Weekly Delivery.
+            Choose A Membership That's Perfect for You!</p>
+          <button type="submit" class="btn border-0 outline-none bg-danger text-white p-2 h5 rounded">SUBSCRIBE
+            &gt;</button>
+        </div>
+      </div>
+    </div>
+    <div class="col-lg-4 mt-md-5 my-5 m-lg-0 ">
+      <h6 class="red text-danger">NEWSLETTER</h6>
```

Comment:
- [ ] Make sure that this title has the correct font-weight. 
This, 'Subscribe', ' Featured Stories', 'Top Story', 'The Debate', and 'Opinion' in special also have the same issue of having a too low font-weight.

Compare your clone and the original archived site and make sure the text styles match.

Reply:
- [ ] Spend more time on the responsiveness of the 'Subscribe' section text on the tablet size:
**Your version**
![Screenshot_2021-01-27 Newsweek(3)](https://user-images.githubusercontent.com/16193212/106048676-f3cb2c80-60c3-11eb-9efb-832621411ddf.png)
**Original version**
![subscribe_and_newsletter_tablet](https://user-images.githubusercontent.com/16193212/106049307-c29f2c00-60c4-11eb-9813-26bff4cd6786.png)

---

- [ ] Spend more time on the responsiveness of the 'Newsletter' section image on the mobile size:
**Your version**
![Screenshot_2021-01-27 Newsweek](https://user-images.githubusercontent.com/16193212/106048684-f594f000-60c3-11eb-8577-c7a0261110b3.png)
**Original version**
![subscribe_and_newsletter_mobile](https://user-images.githubusercontent.com/16193212/106049277-b87d2d80-60c4-11eb-9918-5c2638d1263c.png)